media-sound/murmur/murmur-9999.ebuild

```diff
@@ -93,36 +95,39 @@ src_prepare() {
 	# Adjust systemd service file to our config location #689208
 	sed "s@/etc/${PN}\.ini@/etc/${PN}/${PN}.ini@" \
 		-i scripts/${PN}.service || die
+
+	cmake_src_prepare
 }
 
 src_configure() {
 	myuse() {
 		[[ -n "${1}" ]] || die "myconf: No use flag given."
 		use ${1} || echo "no-${1}"
 	}
-	local conf_add=(
-		no-client
-		$(myuse dbus)
-		$(usex debug 'symbols debug' release)
-		$(myuse ice)
-		$(myuse pch)
-		$(usex zeroconf '' no-bonjour)
+	local mycmakeargs=(
+		"-DBUILD_TESTING=$(usex test)"
+		"-Dclient=OFF"
+		"-Ddbus=$(usex dbus)"
+		"-Dg15=OFF"
+		"-Dgrpc=$(usex grpc)"
+		"-Dice=$(usex ice)"
+		"-Doverlay=OFF"
+		"-Dserver=ON"
+		"-Dzeroconf=$(usex zeroconf)"
```

Comment:
Please also fix the quoting like I did in the mumble ebuild in commit 3629693f926f2a47c4adc08e28987bb265a7aa4a

Reply:
> Looking good. There's only the quoting that needs fixing (see my other comment)

Thanks for the review. Quoting has been adapted and ebuild re-tested